Package.swift

```diff
@@ -12,14 +12,14 @@ let package = Package(name: "Diagnostics",
                         .watchOS(.v6)],
                       products: [
                         // dev .library(name: "DangerDeps", type: .dynamic, targets: ["DangerDependencies"]),
-                        .library(name: "Diagnostics", targets: ["Diagnostics"])
+                        .library(name: "Diagnostics", type: .static, targets: ["Diagnostics"])
                         ],
                       dependencies: [
-                        // dev .package(url: "https://github.com/danger/swift", from: "3.0.0"),
-                        // dev .package(path: "Submodules/WeTransfer-iOS-CI/Danger-Swift")
+                        // dev .package(name: "danger-swift", url: "https://github.com/danger/swift", from: "3.0.0"),
+                        // dev .package(name: "WeTransferPRLinter", path: "Submodules/WeTransfer-iOS-CI/Danger-Swift"),
```

Comment:
<!--
  1 failure:  Collection litera...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Package.swift;
  Line: 19;
-->
- :no_entry_sign: Collection literals should not have trailing commas. (`trailing_comma`)